namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "route53_zone" {
+  name = var.prepare-case-domain
+
+  tags = {
+    business-unit          = var.business-unit
+    application            = var.application
+    is-production          = var.is-production
+    environment-name       = var.environment-name
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure-support
+  }
+}
+
+resource "kubernetes_secret" "route53_zone_sec" {
+  metadata {
+    name      = "route53-zone-output"
```

Comment:
Same here please

Reply:
Does the name here also have to be unique?

Reply:
yes, it does. 
Also, for clarity it is better to have easily identifiable secrets name.

Reply:
OK, thanks. Updated now.